config/initializers/omniauth.rb

```diff
@@ -12,7 +12,7 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
Actually can we fix that cop? I'd generally rather have parenthesis around post-conditionals. I've been bitten at least once by less-than-intuitive precedence when they're removed (because, naturally, rubocop complained about them).